### ISSUE TITLE
KeymanWeb needs to optimise static stores

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#1007]
When compiling a keyboard with repeated platform statements, this generates many repeated stores. For an example, see https://github.com/silcam/Cameroon-Keyboard/blob/BrokenRules/src/km/cameroon_qwerty/source/cameroon_qwerty.kmn

If repeated stores are not used as a variable store, then they should be optimised to a single reference at compile time.

**Comment by erros84** [comment]
Just a note, those platform stores were only needed to avoid KM9's RALT/CAPS limitation. I'm not saying you don't need to fix it, but I'm removing them.

**Comment by mcdurdin** [comment]
Yeah, I appreciate that for this specific keyboard it may not be such an issue in the future. :)

**Comment by MattGyverLee** [comment]
My keyboards no longer "spam" the platform statements. If you were holding this issue open for me, you can probably close it.

**Comment by mcdurdin** [comment]
It's not specific to your keyboards @MattGyverLee; it's a general optimization that we should do one day.
